src/translations/fr/campaign.php

```diff
@@ -15,12 +15,12 @@
     'Plugin Name' => 'Newsletters',
     // -------------------------------------------------------------------------
     '“{name}” Campaigns' => '“{name}” Newsletter',
-    '“{name}” Mailing Lists' => '“{name}” Listes d’envoi',
-    '“Include original headers” must be enabled for each notification type.<br>See this <a href="{url}" target="_blank">article</a> for details on where and how to enable this.' => '',
+    '“{name}” Mailing Lists' => '“{name}” Listes de diffusion',
+    '“Include original headers” must be enabled for each notification type.<br>See this <a href="{url}" target="_blank">article</a> for details on where and how to enable this.' => 'L’option « Include original headers » doit être activée pour chaque type de notification. <br>Voir cet <a href=« {url} » target=« _blank »>article</a> pour plus de détails sur l’endroit et la manière d’activer cette option.',
```

Comment:
Please fix this so that anchor tags remain unchanged, so `<a href="{url}" target="_blank">` should be maintained.